Formula/t/tika.rb

```diff
@@ -19,11 +20,13 @@ class Tika < Formula
   end
 
   def install
+    odie "server resource needs to be updated" if version != resource("server").version
+
```

Comment:
Let's add this to the test block as a check

Reply:
yeah, I will do later in a bit.

Reply:
Now, I see your point, I think we'd. prefer to have odie in the install so that we dont need to wait for the test to check on the resource sync thing. Let me know if that makes sense.

Reply:
This aligns with bunch of build change PRs on the similar resource sync issue